### PR TITLE
[mongodb] Example Tests for MongoDBSource

### PR DESCRIPTION
**Pull request by vajaw**
Example Tests for MongoDBSource

### CODE REVIEW

File: flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/MongoDBSourceExampleTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+package com.ververica.cdc.connectors.mongodb;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+
+import com.ververica.cdc.debezium.JsonDebeziumDeserializationSchema;
+
+/** Example Tests for {@link MongoDBSource}. */
+public class MongoDBSourceExampleTest {
```
Comment (leonardBang) [inline flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/MongoDBSourceExampleTest.java:27]: Thanks @vajaw for the contribution, could you refer `MySqlSourceExampleTest` to make `MongoDBSourceExampleTest` be a unit test without any preparation.

Comment (vajaw) [reply flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/MongoDBSourceExampleTest.java:27]: Ok, i can

Comment (Jiabao-Sun) [reply flink-connector-mongodb-cdc/src/test/java/com/ververica/cdc/connectors/mongodb/MongoDBSourceExampleTest.java:27]: Hi @vajaw, are you going to push a commit to resolve this comment?

1.  Extends `MongoDBTestBase` to setup a test MongoDB instance.
2. Prepare some records by: `String database = executeCommandFileInSeparateDatabase("inventory");`
3. Connect the MongoDB instance then print the data.

**Comment by Jiabao-Sun** [comment]
Closed this because opened a new PR #1388.

**Pull request closed without merging**